### PR TITLE
Docs: update deployment steps

### PR DESCRIPTION
**Pull request by tadhg-ohiggins**
We altered the steps.
Pray that we do not need to
Alter them further.

-----

Describe the current deployment steps in detail now that we appear to have GitHub actions and Terraform working.

### CODE REVIEW

File: docs/deploying.md [hunk 1]
```diff
@@ -53,6 +55,43 @@ cf push -f manifests/manifest-dev.yml
...
+
+#### Deploying to `staging`
+
+1.  Create a pull request using `main` targeting `prod`
+    *   In the GitHub interface, this will mean setting **base** to `prod` and **compare** to `main`.
+    *   The title of the pull request should start with `[YYYY-MM-DD]` and indicate that it’s a merge from `main` to `prod`, for example `2023-04-03 main -> prod`.
+2.  Request reviewers.
+3.  Wait for the checks to pass.
+4.  If checks fail, or if reviewers request changes, something has gone awry. Investigation and/or starting over from a branch and making a PR against `main` may be required.
+5.  Either the reviewer or the submitter can merge the approved pull request.
```
Comment (LindsayYoung) [inline docs/deploying.md:79]: I think only FAC Admins can merge the PR right?

Comment (tadhg-ohiggins) [reply docs/deploying.md]: Corrected, thanks!

File: docs/deploying.md [hunk 2]
```diff
@@ -53,6 +55,43 @@ cf push -f manifests/manifest-dev.yml
...
+
+1.  Create a [new release](https://github.com/GSA-TTS/FAC/releases/new).
+    *   Create a new tag in the form `v1.[YYYYMMDD]` with the current date, such as `v1.20230329`, and use that as the tag for the release.
+    *   **_Remember to set the target to `prod`_**.
+    *   For the moment, check the “Set as a re-release” checkbox. We may change this close to or after MVP.
+3.  Wait for the checks to pass.
+4.  If checks fail, or if reviewers request changes, something has gone awry. Investigation and/or starting over from a branch and making a PR against `main` may be required.
+5.  Verify that the deploy steps all passed.
+6.  After deployment, the changes should be on https://fac-prod.app.cloud.gov/.
+
```
Comment (LindsayYoung) [inline docs/deploying.md:94]:
```suggestion
7.  If any thing was merged directly to `prod`, like a hotfix, merge `prod` back into main.
```

Comment (tadhg-ohiggins) [reply docs/deploying.md:95]: I added an equivalent line, thanks!

**Review by mogul — APPROVED** [review]

**Pull request merged**